### ISSUE TITLE
Can this run whole folders?

### ISSUE THREAD

**Issue by nico-abram** [trixnz/lua-fmt#28]
I cant find any docs, and the only example/s provided in the readme are single file

**Comment by qaisjp** [comment]
No. Use it with the "find" command to apply it to multiple files
